reposilite-frontend/src/components/icons/DownIcon.vue

```diff
@@ -0,0 +1,24 @@
+<!--
+  - Copyright (c) 2021 dzikoysk
+  -
+  - Licensed under the Apache License, Version 2.0 (the "License");
+  - you may not use this file except in compliance with the License.
+  - You may obtain a copy of the License at
+  -
+  -     http://www.apache.org/licenses/LICENSE-2.0
+  -
+  - Unless required by applicable law or agreed to in writing, software
+  - distributed under the License is distributed on an "AS IS" BASIS,
+  - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  - See the License for the specific language governing permissions and
+  - limitations under the License.
+  -->
+
+<template>
+  <svg viewBox="0 0 100 100" width="15" height="15">
+    <defs>
+      <image  width="77" height="51" id="img1" href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAE0AAAAzCAMAAADl/3UzAAAAAXNSR0IB2cksfwAAAD9QTFRFAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAsyWCLwAAABV0Uk5TABCP799vz/+fYIC/8JDQMCBQcKDgzmeILAAAATlJREFUeJyt19GSgyAMBdBYrQHX1nbt/3/r0nFtFRLIRfMYvGdGZSQSrdVc2u5KcF279tIk3abnUA7F3DvVx9yCwZxbUhG3YswewYY1teO+GMT5b2rDbTGA89vUz6ftmCs4v0+tTzzCjJyPU07GTFyCLdyQtg2cgDEPRJ3UL3Eixh3RKC7kORnjkegmr+Q4BeNb2Gx3lNOw+3sHa1w7ydiUw1BuarMYxhUxhDNggVP2ScyZsHCZidOwMb4HC2fGLByAlTkIK3EgluHCG3ugmM71zf7oMGEZrgbTuSoM44oY0dOulTH9Y5iU7dw1ctaZwMTZ5xUDh8xSRQ4azEochhU4FMtyOJbhajCVq8OUEeW3EiOaU2yuxgTuCJZwx7CIO4rtuOPYhjsD+3DnYP/cWVj4p3i9htMwc/0ByI4x8MjWouIAAAAASUVORK5CYII="/>
+    </defs>
+    <use  href="#img1" x="13" y="27" />
```

Comment:
What do you think about replacing this encoded image with this pure SVG icon:

* https://heroicons.dev/?query=arrow (`chevron-dow`)

Reply:
Done, updated the svg icon